static/app/views/performance/trends/chart.tsx

```diff
@@ -209,6 +209,8 @@ export function Chart({
     needsLabel,
     transaction,
     applyRegressionFormatToInterval
+      ? IntervalLineFormat.REGRESSION_TIMESERIES
+      : IntervalLineFormat.DEFAULT
```

Comment:
I actually think we can drop this and just pass `IntervalLineFormat.DEFAULT` here because we're not using this component in more than just Trends anymore, we created our own line chart component so functions were able to pass down their own series data

We can simplify it here or in some cleanup PR

Reply:
After realizing that I was still bringing along those baselines and really only needed the dividing line, I moved all that code out of utils meaning I didn't need to make any changes to `getIntervalLine` anymore.